ui/components/ItemDetail/ItemDetail.tsx

```diff
@@ -209,18 +210,18 @@ const ItemDetail = ({ selectedItem, handleClose, handleSelectItem }) => {
 
       <AideApprentissage item={selectedItem}></AideApprentissage>
 
-      {kind === "peJob" && (
+      {kind === LBA_ITEM_TYPE.OFFRES_EMPLOI_PARTENAIRES && (
         <>
           <DidYouKnow />
           {!buttonJePostuleShouldBeDisplayed(kind, selectedItem) && (
             <GoingToContactQuestion kind={kind} uniqId={getGoingtoId(kind, selectedItem)} key={getGoingtoId(kind, selectedItem)} item={selectedItem} />
           )}
         </>
       )}
-      {kind === "formation" && !buttonRdvShouldBeDisplayed(selectedItem) && (
+      {kind === LBA_ITEM_TYPE.FORMATION && !buttonRdvShouldBeDisplayed(selectedItem) && (
         <GoingToContactQuestion kind={kind} uniqId={getGoingtoId(kind, selectedItem)} key={getGoingtoId(kind, selectedItem)} item={selectedItem} />
       )}
-      {(kind === "lbb" || kind === "lba") && !isCandidatureLba(selectedItem) && (
+      {amongst(kind, [LBA_ITEM_TYPE.RECRUTEURS_LBA]) && !isCandidatureLba(selectedItem) && (
```

Comment:
```suggestion
      {kind === LBA_ITEM_TYPE.RECRUTEURS_LBA && !isCandidatureLba(selectedItem) && (
```
`amongst` ne sert à rien, j'aimerais bien le supprimer du code à terme